src/config.ts

```diff
@@ -283,7 +283,7 @@ export const GAS_SCHEDULE_V0: FeeSchedule = {
     [ERC20BridgeSource.Eth2Dai]: () => 5.5e5,
     [ERC20BridgeSource.Kyber]: () => 8e5,
     [ERC20BridgeSource.Curve]: fillData => {
-        switch ((fillData as CurveFillData).poolAddress.toLowerCase()) {
+        switch ((fillData as CurveFillData).curve.poolAddress.toLowerCase()) {
```

Comment:
We also need to add estimates for `0x93054188d876f558f4a66b2ef1d97d16edf0895b` and `0x7fc77b5c7614e1533320ea6ddc2eb61fa00a9714`. These are about 600k on v0.

Reply:
done